RELEASE.md

```diff
@@ -1,10 +1,22 @@
 ### How to release Rally?
 
-The release process in Rally is automated as much as possible. Suppose you want to release Rally 1.2.0. Then just run `make release release_version=1.2.0 next_version=1.2.1`. The parameter `release_version` refers to the release that is being released and the parameter `next_version` refers to the next development version. 
+The release process in Rally is automated as much as possible.
+It will automatically run all tests (including integration tests), build and upload all artifacts and update the changelog and the list of contributors.
 
-This will automatically run all tests (including integration tests), build and upload all artifacts and update the changelog and the list of contributors.
+### Example: suppose you want to release Rally 2.0.1
```

Comment:
Did you mean to say Rally 2.0.0? The example where the `release` target is invoked suggests that.

Reply:
Typo fixed in 4a0c1db03491289fecd772e5157079f2c8be5c8e